### DISCUSSION TITLE
Cannot Apply for Student Benefits? Looking for another way except webcam

### DISCUSSION THREAD

**Discussion by dcelik81** [community/community#109562] [GitHub Education] [answered]
### Hi everyone,

Hello guys,
I had Github Student Pack when I was a freshman. I was easy, I just have uploaded the document and it was okay. 

But now it wants me to take a photo of the document with webcam or smartphone camera. I tried 2 different phones, feature does not work well. It is not high resolution, and the problem is not my phone. If I try my laptops webcam, then it looks so bad, it is not even HD. Even I tried it and it says failed because computer cannot verify the date and the information on the paper, neither I am. I cannot even read my own document through that trashlike webcam.

![image](https://github.com/community/community/assets/71578903/8be569db-80c7-4ad7-bb16-cb666f4cd6d7)

So is there any other ways to apply for this benefits? I am helpless. Please help me.

**Comment by namith1003** [comment]
ISTG I've just had the same issue, In the end I realized there is no other way to get this done, maybe you could contact support or maybe ask them but I'm not fully sure, best case option you have to get a friend with an amazing smartphone camera and go to really good lighting and try your very best to take a clear photo, it worked for me in that way, my camera could never.

Hopefully this helps answer your question.

**Reply by aleksiye** [reply]
How long did it take for you for it to accept it? No matter what I try it just returns me back to the same page

**Comment by dcelik81** [comment, marked as answer]
I got the student benefits. I had no date on my student id card. That's why I thought it would not be appropriate to scan it but it worked. Thank you guys for the answers.
